src/core/data/index.ts

```diff
@@ -222,6 +227,52 @@ export interface HealthSavingsAccount<DateType = string> {
   qualifiedDistributions: number
 }
 
+export enum IraPlanType {
+  IRA = 'IRA',
+  RothIRA = 'RothIRA',
+  SepIRA = 'SepIRA',
+  SimpleIRA = 'SimpleIRA'
+}
+
+export const IraPlanTypeTexts = {
+  [IraPlanType.IRA]: 'Traditional IRA',
+  [IraPlanType.RothIRA]: 'Roth IRA',
+  [IraPlanType.SepIRA]: 'Simplified employee pension (SEP) IRA',
+  [IraPlanType.SimpleIRA]:
+    'Savings incentive match plan for employees (SIMPLE) IRA'
+}
+
+export type IraPlanName = keyof typeof IraPlanType
+
+// Typescript provides no way to access
+// keys of an interface at runtime.
+export const iraPlanNames: IraPlanName[] = [
```

Comment:
Generally if you need an array of the names of an enum, you can use [enumKeys](https://github.com/ustaxes/UsTaxes/blob/7056ea37276f6d610b9dee5d9b1e7847871be82a/src/core/util.ts#L8).